add-on/panels/test/panel_test.js

```diff
@@ -952,12 +982,10 @@ describe("loop.panel", function() {
     });
 
     it("should call mozL10n.get for room empty strings", function() {
-      createTestComponent();
+      var view = createTestComponent();
 
-      sinon.assert.calledWithExactly(document.mozL10n.get,
-                                     "no_conversations_message_heading2");
-      sinon.assert.calledWithExactly(document.mozL10n.get,
-                                     "no_conversations_start_message2");
+      expect(view.getDOMNode().querySelectorAll(".room-list-empty").length).to.eql(1);
+      expect(view.getDOMNode().querySelectorAll(".panel-text-medium").length).to.eql(2);
```

Comment:
This will change soon as there are some bugs that want this behavior to be changed.